app/models/partners/profile.rb

```diff
@@ -108,5 +115,18 @@ class Profile < Base
     def has_no_social_media?
       website.blank? && twitter.blank? && facebook.blank? && instagram.blank?
     end
+
+    def client_share_total
+      served_areas.sum(&:client_share)
+    end
+
+    def client_share_is_0_or_100
+      # business logic:  the client share has to be 0 or 100 -- although it is an estimate only,  making it 0 (not
+      # specified at all) or 100 means we won't have people overallocating (> 100) and that they think about what
+      # their allocation actually is
+      if client_share_total != 0 && client_share_total != 100
```

Comment:
Can we add a variable here to act as a cache? Otherwise it'll sum the client shares twice instead of once.